metals/src/main/scala/scala/meta/metals/Main.scala

```diff
@@ -14,21 +16,68 @@ import scala.meta.internal.metals.Trace
 import scala.meta.internal.metals.clients.language.MetalsLanguageClient
 import scala.meta.internal.metals.logging.MetalsLogger
 
+import coursierapi.Complete
 import org.eclipse.lsp4j.jsonrpc.Launcher
 
 object Main {
   def main(args: Array[String]): Unit = {
     if (args.exists(Set("-v", "--version", "-version"))) {
+      val api: Complete = coursierapi.Complete
+        .create()
+        .withInput("org.scalameta:mtags")
```

Comment:
One problem I see is that it will not work for snapshot versions of Metals. The alternative would be something similar to my super fun workaround here: https://github.com/scalameta/metals/blob/main/metals/src/main/scala/scala/meta/internal/metals/MtagsResolver.scala#L119